programmers/난이도별/level01.약수의_합/wander0220.js

```diff
@@ -0,0 +1,9 @@
+function wander0220(n) {
+    var sum = 0;
+    
+    for(var i = 1 ; i <= n ; i++){
+        if(n%i==0) sum+=i;
+    }
+    
+    return sum;
```

Comment:
```suggestion
    for(var i = 1 ; i <= n/2 ; i++){
        if(n%i==0) sum+=i;
    }
    
    return sum+n;
```
이렇게 하면 더 효율적일것 같아요!!

Reply:
오~~ 감사합니다!